src/app/Main/Body/InfoPage/Amrs/InfoPage-AmrsDialog.component.tsx

```diff
@@ -50,7 +50,7 @@ function createTableRowCells(row: AmrsTableData): JSX.Element[] {
             {row.amrSubstance}
         </TableCell>
     );
-    Object.values(row.valuesPerYearObject).forEach((yearValues) => {
+    Object.values(row.concentrationList).forEach((concentrations) => {
```

Comment:
So this can be simplified now: `row.concentrationList.foreach(concentration => {`